### PR TITLE
Feed Skeleton Implemented

### PR DESCRIPTION
**Pull request by 7ingyu**

### CODE REVIEW

File: parley/src/App.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+/* Core CSS required for Ionic components to work properly */
+import '@ionic/react/css/core.css';
+
+/* Basic CSS for apps built with Ionic */
+import '@ionic/react/css/normalize.css';
+import '@ionic/react/css/structure.css';
+import '@ionic/react/css/typography.css';
+
+/* Optional CSS utils that can be commented out */
```
Comment (evankatz14) [inline parley/src/App.tsx:14]: at the end of the project let's just remember to comment out or remove the imports that we aren't using

File: parley/src/components/ExploreContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import './ExploreContainer.css';
```
Comment (evankatz14) [inline parley/src/components/ExploreContainer.tsx:1]: similarly, at some point we will want to clear out the unused boiler plate

File: parley/src/components/Header/Header.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      </IonThumbnail>
+      <h1>PARLEY</h1>
+      <ThemeToggle />
+      <IonButton icon-only className="searchbtn" slot="end" href="/search">
+        <IonIcon className="searchicon" icon={searchOutline}/>
+      </IonButton>
+      <IonItem className="profilebtn" lines="none" slot="end" href="/profile">
+        <IonChip>
+        <IonAvatar>
+        <img alt={'Your Profile'} src={user.profile_img} />
```
Comment (evankatz14) [inline parley/src/components/Header/Header.tsx:21]: maybe the alt tag should say 'Your Profile Pic'? Or 'Profile Pic with link to profile page'?

File: parley/src/components/List/List.tsx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+          element.open();
+        }
+      })
+  }
+
+  return (
+    <IonCard>
+      <IonItem onClick={() => setFold(string)}>
+        <IonCardTitle>
+          {isStreaming ? <h2>Live Streams</h2>: <h2>Recordings</h2>}
```
Comment (evankatz14) [inline parley/src/components/List/List.tsx:28]: very small thing, but lets make sure to have a space before the colon for readability => 
`{isStreaming ? <h2>Live Streams</h2> : <h2>Recordings</h2>}`

File: parley/src/components/List/List.tsx [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+        </IonCardTitle>
+          {unfolded ? <IonIcon icon={chevronUpOutline} slot="end"></IonIcon>: <IonIcon icon={chevronDownOutline} slot="end"></IonIcon>}
+      </IonItem>
+    {unfolded ? <IonList>
```
Comment (evankatz14) [inline parley/src/components/List/List.tsx:32]: the format I have seen for ternaries in react is like so:
```
{unfolded ? (
    *code
) : (
    *code
)}
```

**Review by evankatz14 — APPROVED** [review]

File: parley/src/components/Header/Header.tsx (hunk 3 above)
Comment (7ingyu) [reply parley/src/components/Header/Header.tsx:21]: According to Ionic, adding image/pic is redundant as screen readers can already detect that it's an image. Same with links, I think.

**Pull request merged**